go/libkb/features.go

```diff
@@ -30,15 +30,6 @@ func (set FeatureFlags) Admin() bool {
 	return false
 }
 
-func (set FeatureFlags) UseEphemeral() bool {
-	for _, f := range set {
-		if f == Feature("use-ephemeral") {
-			return true
-		}
-	}
-	return false
-}
```

Comment:
@maxtaco was talking today about wanting to disable some EK stuff for the "oneshot" devices he's working on, where you type in your paper key in a docker image or whatever. Would we want to keep `UseEphemeral`, and add that sort of logic to it? Or would there be a better place?

Reply:
i added this as an extra feature flag so we did store keys under the `admin` flag and then have a headache with other devs having to blow away bad files. going forward we should just use the `admin` flag so it's tested more widely before we fully release